templates/shared_services/gitea/terraform/gitea-webapp.tf

```diff
@@ -267,6 +267,7 @@ resource "azurerm_role_assignment" "gitea_acrpull_role" {
 resource "null_resource" "webapp_vault_access_identity" {
```

Comment:
Look like this is now supported by the terraform provider: https://github.com/hashicorp/terraform-provider-azurerm/issues/13388